lib/sqlalchemy/orm/unitofwork.py

```diff
@@ -13,11 +13,12 @@
 
 """
 
+import collections
```

Comment:
**mike bayer** (zzzeek) wrote:

so there is a module called sqlalchemy.orm.collections.   so to make this work in Python 2.7, you need to add "from __future__ import absolute_import" as the first import in the file.